apps/mobile-alphabet/src/app/Components/Menu.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import { Text, View } from 'react-native';
+
+interface AlphabetData {
+    data: {
+        name: string;
+        name_english: string;
+        poster: {
+            name: string;
+            url: string;
+        };
+    };
+}
+
+export function MenuScreen() {
+    const [alphabetData, setAlphabetData] = useState<AlphabetData | null>(null);
+
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const response = await fetch('http://10.0.2.2:3131/api/games/alphabet', {
+                    mode: 'cors',
+                });
+                setAlphabetData(await response.json());
+            } catch (error) {
+                console.error('Error fetching data:', error);
```

Comment:
You need to update the component state with the error and render an error message.